### PR TITLE
Fixed weather$time_hour timezone and added to documentation for fligh…

### PR DESCRIPTION
**Pull request by rudeboybert**
This fixes Issue #19 and adds timezone clarification to `flights` and `weather` man pages. Note: package needs to be cleaned and `weather.rda` file needs to be regenerated, since the code in `data-raw/weather.R` did not work for me.

### CODE REVIEW

**Comment by rmcd1024** [comment]
Aviation weather data is typically time-stamped in GMT. My suggestion would be to document the time-zone difference (I don't believe this is currently mentioned anywhere) and leave the correction as an exercise for the user.  

Separate issue: does your patch account for daylight savings time? I would have expected a time zone entry like "America/New_York" rather than "EST".

**Comment by rudeboybert** [comment]
Normally I would be fine with just documenting this as well, but this does come at the expense of being able to conveniently `left_join(flights, weather, by="time_hour")` instead of the more unwieldy `left_join(flights, weather, by=c("month", "day", "hour", "origin"))`

File: data-raw/weather.R [hunk 1]
```diff
@@ -66,8 +66,11 @@ weather <- raw %>%
   ungroup() %>%
   filter(!is.na(month)) %>%
   mutate(
-    time_hour = ISOdatetime(year, month, day, hour, 0, 0)
-  )
+    time_hour = ISOdatetime(year, month, day, hour, 0, 0),
```
Comment (hadley) [inline data-raw/weather.R]: Are you sure you can't just set `tz` here? 

Your current code doesn't feel right to me - you shouldn't have to both force the timezone and then add a manual offset

**Comment by rudeboybert** [comment]
Agreed. @ltierney suggested a much cleaner fix in #19 but there are still some unresolved timezone formatting issues to address there first.

**Pull request closed without merging**